.circleci/config.yml

```
@@ -159,6 +208,9 @@ commands:
           at: .
 
 parameters:
+  go_base_url:
+    type: string
```

nitpick: can we `enum` this?
```suggestion
    type: enum
    enum: ["https://go.dev/dl/", "https://aka.ms/golang/release/latest/"]
```

---

We already have this in the python script, I don't feel comfortable to have it in both places. where do you think it makes most sense?

---

Should we use `latest` or pin it? There's a new 1.21 release for microsoft go which seems to be "safer" to use. https://github.com/microsoft/go/blob/microsoft/release-branch.go1.21/docs/go1.21.md

---

There is a pipeline parameter that is being used to specify the go version, which we should continue. The 'latest' in the URL might be incorrect, at least it is not intended to use latest. Let me check.

---

Okay, 'latest' is actually part of the base url, there are the following download links
* https://aka.ms/golang/release/latest/go1.20.linux-amd64.tar.gz
* https://aka.ms/golang/release/latest/go1.19.linux-amd64.tar.gz
* https://aka.ms/golang/release/latest/go1.21.linux-amd64.tar.gz

---

latest probably relates to the latest version of the fork. It might make sense to pin this.